src/taipy/core/scenario/scenario.py

```diff
@@ -382,24 +437,26 @@ def is_deletable(self) -> bool:
 
         return tp.is_deletable(self)
 
-    def __get_pipelines(self):
-        pipelines = {}
-        pipeline_manager = _PipelineManagerFactory._build_manager()
+    # TODO: TBD
+    # def __get_pipelines(self):
+    #     pipelines = {}
+    #     pipeline_manager = _PipelineManagerFactory._build_manager()
 
-        for pipeline_or_id in self._pipelines:
-            p = pipeline_manager._get(pipeline_or_id, pipeline_or_id)
+    #     for pipeline_or_id in self._pipelines:
+    #         p = pipeline_manager._get(pipeline_or_id, pipeline_or_id)
 
-            if not isinstance(p, Pipeline):
-                raise NonExistingPipeline(pipeline_or_id)
-            pipelines[p.config_id] = p
-        return pipelines
+    #         if not isinstance(p, Pipeline):
+    #             raise NonExistingPipeline(pipeline_or_id)
+    #         pipelines[p.config_id] = p
+    #     return pipelines
 
     @classmethod
-    def _to_model(cls, entity) -> _ScenarioModel:
+    def _to_model(cls, entity: Scenario) -> _ScenarioModel:
```

Comment:
is the method still used in 3.0?

Reply:
I believe it's a no. But I will leave the decision to Joao when he works on the removal of the old repositories and I can just merge the changes from his part :D